### PR TITLE
replaced references of the generator app with the generator URL directly

### PR DESCRIPTION
**Pull request by tbsbdr**
- Replaced references to the generator app for Admins with the "raw" generator url, to enable users to integrate ownCloud into MS Teams right now.

**Context:** The app submission of the generator app for Admins to the MS Teams App Store is still delayed. Meanwhile we don't want to prevent users from integrating ownCloud into MS Teams, because this is actually possible without the official generator app for Admins. It can be directly done via https://msteamsgen.owncloud.com/. (the generator app for Admins actually just encapsulates the generator on https://msteamsgen.owncloud.com/ )

### CODE REVIEW

**Comment by mmattel** [comment]
This needs a forward port to master.

@tbsbdr for the next PR, please use as base branch master, we will backport it to all relevant branches. The base for docs is always master exept there is a PR which fixes a branch specific issue only...

File: modules/admin_manual/pages/configuration/integration/ms-teams.adoc [hunk 1]
```diff
@@ -6,10 +6,11 @@
...
 
 == Introduction
 
-If you're using Microsoft Teams in your organization or for private purposes, you will likely want to access your ownCloud installation from your Microsoft Teams account. For this purpose, we created the {appsource-url}[Microsoft AppSource] app *ownCloud Generator for Admins* (expected to be available from end of August 2021) with which you can generate a customized Microsoft Teams app for your users accessing your ownCloud services. Each ownCloud domain accessed requires a separate generated Microsoft Teams app for your users. The generated app will be available in your organization's app catalog.
+If you're using Microsoft Teams in your organization or for private purposes, you will likely want to access your ownCloud installation from your Microsoft Teams account. For this purpose, we created the {msteams-generator-url}[ownCloud Generator for Admins] with which you can generate a customized Microsoft Teams app for your users accessing your ownCloud services. Each ownCloud domain accessed requires a separate generated Microsoft Teams app for your users. The generated app will be available in your organization's app catalog.
```
Comment (mmattel) [inline modules/admin_manual/pages/configuration/integration/ms-teams.adoc:13]: > The generated app will be available in your organization's app catalog.

This reads as there is an automatism behind and conflicts with the statement:
"Go to the app section of Microsoft Teams and upload the generated zip file..."

Needs an improved text

Comment (tbsbdr) [reply modules/admin_manual/pages/configuration/integration/ms-teams.adoc]: good point, changed it: https://github.com/owncloud/docs/pull/4045/commits/d56a6ce52f6acb8e235c5c5eddd53ba7b9492e17

**Review by mmattel — APPROVED** [review]

**Comment by tbsbdr** [comment]
> This needs a forward port to master.
> 
> @tbsbdr for the next PR, please use as base branch master, we will backport it to all relevant branches. The base for docs is always master exept there is a PR which fixes a branch specific issue only...

oh yes, will do. thanks for the notice!

**Comment by phil-davis** [comment]
PR #4051 has already done the forward-port.

**Pull request merged**